plugins/alert-spell/alert-spell.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const settings = require('./alert-spell.json');
+
+module.exports = {
+  name: 'alert-spell', // Name of the Plugin
+  description:
+    'Sends a message to chat of new Spell. Saves latest spell caster to text file for obs&slobs.', // Description
+  chat: false, // Defines this as a Chat Command
+  event: true, // Is this a Event?
+  type: 5, // Type Event
+  command: '', // This is the Command that is typed into Chat!
+  permissions: [], // This is for Permissisons depending on the Platform.
+  alias: [], // Alias commands that preform interesting things.
+  cooldown: 0, // this is Set in Seconds, how long between the next usage of this command.
+  settings: false, // Defining this as false will load the Settings file for this Plugin when the system loads this plugin.
+  credits: `New Spell system by Krammy. Original by Bioblaze Payne.`, // MAKE SURE YOU FILL THIS IN GOD DAMNIT!
+  execute(client, data) {
+    client.sendMessage(`Thanks @${data.user} for the casting a spell <3`);
+
+    if (settings.output)
+      fs.writeFile('./plugins/alert-spell/latest-spell.txt', data.user, (err) => {
```

Comment:
couldn't we save the Spell they are casting, as well as the # they cast, and if it was Elixir or Mana?

Reply:
In a different file yes. As this file is being used just to save who casted it for on-screen showing in OBS and Slobs no.